enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DefaultControllerEnricher.java

```diff
@@ -86,11 +90,11 @@ public DefaultControllerEnricher(EnricherContext buildContext) {
     @Override
     public void addMissingResources(KubernetesListBuilder builder) {
         final String name = getConfig(Config.name, MavenUtil.createDefaultResourceName(getProject()));
-        ResourceConfig config =
-            new ResourceConfig.Builder()
-                .controllerName(name)
-                .imagePullPolicy(getConfig(Config.pullPolicy))
-                .build();
+        ResourceConfig config = new ResourceConfig.Builder()
+                    .controllerName(name)
+                    .imagePullPolicy(getConfig(Config.pullPolicy))
+                    .withReplicas(Integer.parseInt(getConfig(Config.replicaCount)))
```

Comment:
There's a `Configs.asInt()` which you can use to get an int from configuration. For consistencies sake it would make sense to take this.